### DISCUSSION TITLE
PrimeNG SlideMenu not updating when items change

### DISCUSSION THREAD

**Discussion by kencodedit** [primefaces/community#2429] [PrimeNG]
**TLDR: Prime ng Slide menu does not work for dynamic data rendering**
I have a relatively simple use case for the slideMenu, when i click on it, i get a list of items, and when i click any of the list of items, i make a REST GET call and return some items which i intend to display for each item clicked. 

However, the slideMenu briefly display them and remove them. Can someone provide some clarity on this please, or is the slideMenu not sufficient enough to cope with dynamic data rendering.

**Comment by ibrahimkelly** [comment]
> Slide menu do

Sidebar <a href="https://primeng.org/sidebar">PrimeNg Sidebar</a> or Side menu?
Thanks

**Comment by kencodedit** [comment]
PrimeNg Slide Menu: https://www.primefaces.org/primeng-v14-lts/slidemenu
